dist/index.html

```diff
@@ -0,0 +1,27 @@
+<!-- doctype html -->
+<html ng-app="app">
```

Comment:
Why are we still viewing/versioning changes in dist files? or is it just for index?

Reply:
according to .gitignore file: `!dist/index.html`, we're ignoring all dist/ folder except this file